scripts/build.py

```diff
@@ -27,12 +27,12 @@
 import subprocess
 import threading
 
-from core import utils
-from scripts import common
-from scripts import servers
+from scripts import common # isort:skip pylint: disable=wrong-import-position
+from core import utils # isort:skip pylint: disable=wrong-import-position
+from scripts import servers # isort:skip pylint: disable=wrong-import-position
 
-from typing import Deque, Dict, List, Optional, Sequence, TextIO, Tuple
-from typing_extensions import TypedDict
+from typing import Deque, Dict, List, Optional, Sequence, TextIO, Tuple # isort:skip pylint: disable=wrong-import-position
+from typing_extensions import TypedDict # isort:skip pylint: disable=wrong-import-position
```

Comment:
I used the pragma to skip the wrong import order lint error. Not sure if I am using the pragma correctly here though, I had to add these to every import statement for the lint script ignore the error, even though only lines 30 and 31 have the wrong order issue.

Reply:
Can you please add the TODO comment as we do in other places where we move `from scripts import common`?

Reply:
Done